flint.ui/src/components/Slider/Slider.vue

```diff
@@ -44,19 +44,29 @@ export default {
 :deep(.ant-slider:hover .ant-slider-rail) {
   background-color: theme('colors.earth');
 }
+:deep(.ant-slider:hover .ant-slider-handle) {
+  border-color: theme('colors.earth');
+}
+
 :deep(.ant-slider-track) {
-  height: 2px;
+  height: 5px;
   background-color: theme('colors.earth');
+  border-radius: 5px;
 }
 :deep(.ant-slider-rail) {
-  height: 2px;
-  background-color: theme('colors.earth');
+  height: 5px;
+  background-color: gray;
+  border-radius: 5px;
 }
 :deep(.ant-slider-handle) {
-  background-color: theme('colors.earth');
+  background-color: rgb(82, 77, 77);
   border: none;
-  width: 14px;
-  height: 14px;
-  margin-top: -6px;
+  width: 24px;
+  height: 24px;
+  margin-top: -10px;
+  border: 3px;
+  border-style: solid;
+  border-color: white;
+  box-shadow: 2px 2px 5px #aaaaaa;
 }
```

Comment:
We want the `ant-design-vue` dependency of the Slider component completely gone. And we want to use our own<sup>1</sup> slider component.

Please refer to other similar, merged PRs for hints, if you need them.

[1]: https://moja-global-ui-library.vercel.app/?path=/story/components-slider--primary

Reply:
Thank you for the feedback, I've made some changes to the code but it's breaking now, could you please look into it and tell me what i'm doing wrong

Reply:
@ktsrivastava29, Thank you for your efforts but I am not able to see the slider on the screen. I would request you to check back in the UI-Library's documentation once again to find out what is missing. If you can't figure out the problem then please do let us know, and I will check your code on my system.

Reply:
@Janvi-Thakkar ma'am I went through the [documentation](https://moja-global-ui-library.vercel.app/?path=/story/components-slider--primary), i feel i'm missing out something, I have mailed at info@moja.global regarding adding me to the slack community so that i can actively ask and help resolve queries,  but I'm yet to receive any response. It would be kind of you if you could look into the code and help me resolve the issue.